app/views/home/index.html.erb

```diff
@@ -25,10 +25,12 @@
       </thead>
       <tbody>
       <% User.all.each do |user| %>
```

Comment:
The database shouldn't be queried from a view. So one could set `@users = User.where.not(id: current_user.id)` in the index action of the controller and would not need an if here then.

Reply:
we applied it